### PR TITLE
Register ecoplayer07.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Register ecoplayer07.is-a.dev with A record pointing to 41.46.72.138.

### CODE REVIEW

File: domains/ecoplayer07.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "crazyguy66",
+           "email": "kareemehab910@gmail.com",
+           "discord": "962048694293762108"
+        },
+    
+        "record": {
+            "A": ["41.46.72.138"]
```
Comment (CuteDog5695) [inline domains/ecoplayer07.json:9]: cant access website

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/ecoplayer07.json (hunk 1 above)
Comment (crazyguy66) [reply domains/ecoplayer07.json:9]: its open wdym

Comment (CuteDog5695) [reply domains/ecoplayer07.json:9]: i cant visit website, are you sure its the correct IP or is website in a port ip?

Comment (crazyguy66) [reply domains/ecoplayer07.json:9]: isnt the ip supposed to be ipv4?

**Pull request closed without merging**